controls/roles/manage-service/molecule/ssv-lighthouse/converge.yml

```diff
@@ -45,22 +28,35 @@
                 - 0.0.0.0:13000:13000/tcp
               env:
                 CONFIG_PATH: /data/config.yaml
-                CONFIG: |
-                  eth2:
-                    Network: "prater"
-                    BeaconNodeAddr: "http://stereum-{{ beacon_service }}:5052"
-                  eth1:
-                    ETH1Addr: "ws://10.10.0.2:8545"
-                    RegistryContractAddr: "0x687fb596F3892904F879118e2113e1EEe8746C2E"
-                  OperatorPrivateKey: "{{ ssv.sk }}"
-                  global:
-                    LogLevel: "debug"
-                  MetricsAPIPort: 15000
-              command: bash -c "touch /data/config.yaml &&
-                                echo \"$CONFIG\" > /data/config.yaml &&
-                                make BUILD_PATH=/go/bin/ssvnode start-node &&
+              command: bash -c "make BUILD_PATH=/go/bin/ssvnode start-node &&
                                 docker logs ssv_node"
               entrypoint: []
               user: "2000"
               volumes:
                 - "/opt/app/services/{{ ssv_service }}/data/blox/ssv:/data"
+
+    # ssv operator keys
+    - block:
+      - name: "Include ssv-key-generator"
+        include_role:
+          name: "ssv-key-generator"
+      vars:
+        ssv_key_service: "{{ ssv_service }}"
+
+    - name: Get configuration's content in variable
+      slurp:
+        src: "/etc/stereum/services/{{ ssv_service }}.yaml"
+      register: config
+      become: yes
+
+    - name: Read service's configuration
+      set_fact:
+        config_content: "{{ config.content | b64decode | from_yaml }}"
+
+    - name: Write "sk" into SSV-Service's config
+      ansible.builtin.lineinfile:
+        path: "/opt/app/services/{{ ssv_service }}/data/blox/ssv/config.yaml"
+        regexp: '^OperatorPrivateKey:*'
+        insertafter: '^OperatorPrivateKey:*'
```

Comment:
`insertafter` doesn't delete the regexp line, does it? Then there would be 2 lines with `OperatorPrivateKey:`

Reply:
actually it works as its expected. But you right the line `insertafter` is unneccesary. Because `line` include the `OperatorPrivateKey:` and I've just tested it without line `insertafter` and it works well. So the line `insertafter` is removed.